tests/regression/lp-1852361/task.yaml

```diff
@@ -1,5 +1,22 @@
 summary: regression test for https://bugs.launchpad.net/snapd/+bug/1852361
 
+details: |
+    Snapd attempts to update preserved mount namespaces when the per-snap mount
```

Comment:
This it the same description than tests/regression/lp-1844496/task.yaml

Reply:
is this one cased by the same issue?

Reply:
Yes, they are all cross-linked in the last paragraph.